src/main/java/com/google/sps/servlets/UsersServlet.java

```diff
@@ -32,7 +32,7 @@
 import com.google.sps.servlets.User;
 
 /** Servlet that holds the users on this WebApp */
-@WebServlet("/users")
+@WebServlet("/listUsers")
```

Comment:
why is this being changed? i think it will affect some part of the front end since the HttpClient in makes GET requests to /users

Reply:
I changed it to /listUsers just because Xi recommended it in one of our meetings. The design doc also has it named as listUsers now so I was just reflecting the change into the code.